starlite/kwargs.py

```diff
@@ -114,40 +164,38 @@ def create_for_signature_model(
             for key in dependencies
             if key in signature_model.__fields__
         }
-        expected_path_parameters: Set[ParameterDefinition] = set()
-        expected_header_parameters: Set[ParameterDefinition] = set()
-        expected_cookie_parameters: Set[ParameterDefinition] = set()
-        expected_query_parameters: Set[ParameterDefinition] = set()
 
         ignored_keys = {*RESERVED_KWARGS, *[dependency.key for dependency in expected_dependencies]}
```

Comment:
This can be done without having to build the list:

```python
ignored_keys = {*RESERVED_KWARGS, *(dependency.key for dependency in expected_dependencies)}
```

Reply:
Fair point